src/Silk.Data/Models/Tag.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Silk.Data.Models
+{
+    public class Tag
+    {
+        public int Id { get; set; }
+        public int Uses { get; set; }
+        
+        public string Name { get; set; }
+        public string Content { get; set; }
+        
+        public DateTime CreatedAt { get; set; }
+        
+        public ulong OwnerId { get; set; }
+        public ulong GuildId { get; set; }
+        
+        public int? OriginalTagId { get; set; }
+        public Tag? OriginalTag { get; set; }
```

Comment:
You diddddddd.... whyyyyyyyy?????? Yes, you need to split this into two tables. One for aliases, one for all the tag info. And you should define the configuration either using the Fluent API or with the Attributes. Take your pick cuz this is nassssstttyyyyy.

Reply:
1. I initially had two tables. It didn't work out. 
2. Tags and aliases share so much information I don't see the point in making another model and table for them